src/Command/AbstractCommand.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * This file is part of the prooph/micro.
+ * (c) 2017-2017 prooph software GmbH <contact@prooph.de>
+ * (c) 2017-2017 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\Micro\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Yaml\Yaml;
+
+abstract class AbstractCommand extends Command
+{
+    protected function getRootDir(): string
```

Comment:
The root dir for a php process is always `pwd`. So this method could be removed.
If the user wants to provide another dir, a cli option should be used.
Maybe the user should get informed where the docker-compose.yml file would be stored before executed.